actions/filter-files/index.js

```diff
@@ -11,15 +13,24 @@ const parseList = (raw) => {
     return raw.split("\n").map((item) => item.trim());
 };
 
-module.exports = ({extensionsRaw, exactFilesRaw, inputFiles, invert, core}) => {
+module.exports = ({
+    extensionsRaw,
+    exactFilesRaw,
+    globsRaw,
+    inputFiles,
+    invert,
+    core,
+}) => {
     const extensions = parseList(extensionsRaw);
     const exactFiles = parseList(exactFilesRaw);
+    const globMather = picomatch(parseList(globsRaw));
```

Comment:
`picomatch` can handle an array of glob expressions, and an empty array doesn't match anything.